### PR TITLE
V0 Stress Test

### PR DESCRIPTION
**Pull request by lalexgap**
Adds a simple stress test with one `ping`, one `pong` and 100 different channels.

`ping` is called on each channel "concurrently".

### CODE REVIEW

File: packages/server-wallet/package.json [hunk 1]
```diff
@@ -80,6 +80,7 @@
     "start:watch": "nodemon --watch 'src/**/*' -e ts,tsx --exec ts-node ./src/start-dev-server.ts | npx pino-pretty",
     "test": "yarn jest -c ./jest/jest.config.js --runInBand",
     "test:ci": "yarn test --bail",
-    "test:e2e": "SERVER_DB_NAME=ping yarn jest -c ./jest/jest.e2e.config.js"
+    "test:e2e": "SERVER_DB_NAME=ping yarn jest -c ./jest/jest.e2e.config.js",
+    "test:stress": "NODE_ENV=development SERVER_DB_NAME=ping yarn ts-node ./stress-test/one-ping-one-pong.ts"
```
Comment (snario) [inline packages/server-wallet/package.json:84]: Flagging for later, CircleCI expects `NODE_ENV` to be `test`

File: packages/server-wallet/stress-test/one-ping-one-pong.ts [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+    await Channel.bindKnex(knexPong)
+      .query()
+      .insert([{...seed, signingAddress: pong.signingAddress}]);
+    channelIds.push(seed.channelId);
+  }
+  return channelIds;
+}
+
+// eslint-disable-next-line @typescript-eslint/explicit-function-return-type
+(async function() {
```
Comment (snario) [inline packages/server-wallet/stress-test/one-ping-one-pong.ts:44]:
```suggestion
(async function(): Promise<void> {
```

**Comment by lalexgap** [comment]
> It looks like the pong server is not killed at the end of the stress test.

Fixed b939bc0

**Comment by lalexgap** [comment]
Closing in favour of #2418

**Pull request closed without merging**